Sources/Controllers/2.0/Settings/OAImportDuplicatesViewController.mm

```diff
@@ -0,0 +1,518 @@
+                NSString *profileName = modeBean.userProfileName;
+                if (!profileName || profileName.length == 0)
+                {
+                    OAApplicationMode* appMode = [OAApplicationMode valueOfStringKey:modeBean.stringKey def:nil];
+                    profileName = appMode.name;
+                }
+                item[@"label"] = profileName;
+                NSString *routingProfile = @"";
+                NSString *routingProfileValue = modeBean.routingProfile;
+                if (routingProfileValue && routingProfileValue.length > 0)
+                {
+                    try
+                    {
+                        routingProfile = [OARoutingProfileDataObject getLocalizedName: [OARoutingProfileDataObject getValueOf: [routingProfileValue upperCase]]];
+                        routingProfile = [routingProfile capitalizedString];
+
+                    } catch (NSException *e)
+                    {
+                        routingProfile = [routingProfileValue capitalizedString];
+                        NSLog(@"Error trying to get routing resource for %@ \n %@ %@", routingProfileValue, e.name, e.reason);
+                    }
+                }
+                if (!routingProfile || routingProfile.length == 0)
+                    item[@"description"] = @"";
+                else
+                    item[@"description"] = [NSString stringWithFormat:OALocalizedString(@"ltr_or_rtl_combine_via_colon"), OALocalizedString(@"nav_type_hint"), routingProfile];
+                
+                item[@"icon"] = [UIImage imageNamed:modeBean.iconName];
+                item[@"iconColor"] = UIColorFromRGB(modeBean.iconColor);
+                item[@"cellType"] = kMenuSimpleCell;
+            }
+            else if ([currentItem isKindOfClass:OAQuickAction.class])
+            {
+                OAQuickAction *action = (OAQuickAction *)currentItem;
+                item[@"label"] = [action getName];
+                item[@"icon"] = [UIImage imageNamed:[action getIconResName]];
+                item[@"description"] = @"";
+                item[@"cellType"] = kTitleTwoIconsRoundCell;
+            }
+            else if ([currentItem isKindOfClass:OAPOIUIFilter.class])
+            {
+                OAPOIUIFilter *filter = (OAPOIUIFilter *)currentItem;
+                item[@"label"] = [filter getName];
+                NSString *iconRes = [filter getIconId];
+                item[@"icon"] = [UIImage imageNamed: (![iconRes isEqualToString:@"0"] ? iconRes : @"ic_custom_user")]; // check this
+                item[@"description"] = @"";
+                item[@"cellType"] = kTitleTwoIconsRoundCell;
+            }
+            else if ([currentItem isKindOfClass:OASQLiteTileSource.class]) //ITileSource ???
+            {
+                //item[@"label"] = ((OASQLiteTileSource *)currentItem).name; // ?? find how to correctly extract name
+                item[@"label"] = @"Faked name"; // ???
```

Comment:
Do not include code like this into pull requests that are supposed to be accepted into production. You can simply comment out the whole code fragment and return to it later.